twentytwenty-blocks/experimental-theme.json

```diff
@@ -0,0 +1,13 @@
+{
+	"color": {
+		"primary": "#cd2653",
```

Comment:
As per the current changes at https://github.com/WordPress/gutenberg/pull/20530 primary should be removed.